apisix/plugins/log-rotate.lua

```diff
@@ -265,6 +265,22 @@ local function rotate_file(files, now_time, max_kept)
     end
 end
 
+local function get_custom_logfile_name()
+    local custom_file_name = core.config.local_conf()
+
+    local new_error_log = core.table.try_read_attr(local_conf, "nginx_config", "error_log")
+    local new_access_log = core.table.try_read_attr(local_conf, "nginx_config", "http", "access_log")
+
+    if new_access_log then
+        DEFAULT_ACCESS_LOG_FILENAME = new_access_log
```

Comment:
I think we should not use the `DEFAULT_ACCESS_LOG_FILENAME ` and `DEFAULT_ERROR_LOG_FILENAME`, like this
```
local access_log_filename = new_access_log or DEFAULT_ACCESS_LOG_FILENAME 
```

Reply:
Done